packages/cli/src/commands/generate.ts

```diff
@@ -173,18 +173,27 @@ export class Generate extends Command {
     }
   }
 
-  getModelNameAndContext(modelName: string, context?: string): {model: string; context: string} {
+  getModelNameAndContext(modelName: string, context?: string): {model: string; context?: string} {
```

Comment:
Found a bug here while testing the different input values.
Passing not setting the `context` was leading to `contextSegments.= ''` here. However, if we call `path` with empty string we gotta a wrong path:
```ts
console.log(path.join(''))
> .
```
So I noticed a wrong it would generate the wrong import on the newly `__importModelNames` I've added. Since the `options.context` passed down to `generators` would be `.`. That potentially would affect other parts of the code as well.
<img width="890" alt="Screen Shot 2020-06-12 at 20 49 15" src="https://user-images.githubusercontent.com/3356720/84496510-68e22100-acf0-11ea-99d0-21e6abb8691b.png">